gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
```

Comment:
Just checking: normally when we update gradle, a new `gradlew` file is generated too which inflates the diff.

Did you update with a command like this?

`./gradlew wrapper --gradle-version 7.5.1`

Reply:
No, I just updated the properties file directly. I've actually never done it the way you proposed. Is that preferable? Sounds like it

Reply:
Updated

Reply:
Yeah, I'm not entirely sure what _just_ updating this file does, but Gradle docs say to use the wrapper to upgrade versions here: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Thanks!

Reply:
This is fine since you guys all literally work there but since open source, you might want to make sure you are actually validating changes to the gradle wrapper in case you ever get MRs for it in the future

example if validating on GitHub actions:
https://docs.gradle.org/current/userguide/gradle_wrapper.html#wrapper_checksum_verification

If using another CI/CD on prem, you can still make the call to gradle to get the checksums and check against those. A bit of a self-plug but I wrote an article about it with an example of what it guards against (although it's a pretty trivial example):
https://jdvp.me/articles/Gradle-Wrapper-Validation

Also some of the gradle files don't change between releases. For example the wrapper file itself stayed the same between 7.1 and 7.3.2

Reply:
That makes a lot of sense. Thanks @jdvp